src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,13 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+
+        if (!email.equals(null) && !password.equals(null)
+                && userService.findByEmail(email).getEmail().equals(email)
+                && userService.findByEmail(email).getPassword().equals(password)) {
```

Comment:
`any.equals(null) ` **olways** false
do not call `findByEmail `more than 1 time